src/main/java/com/mongodb/hibernate/jdbc/MongoDatabaseMetaData.java

```diff
@@ -67,12 +67,12 @@ public String getDriverVersion() {
 
     @Override
     public int getDriverMajorVersion() {
-        return 1;
+        return 1; // to be implemented in scope of https://jira.mongodb.org/browse/HIBERNATE-34
     }
 
     @Override
     public int getDriverMinorVersion() {
-        return 0;
+        return 0; // // to be implemented in scope of https://jira.mongodb.org/browse/HIBERNATE-34
```

Comment:
Instead of leaving arbitrary-form comments, and not having the corresponding tickets refer back to them, we should to what we agreed to in the Java driver repo:

- The comments are `// TODO-<ticket number> <any additional text>`, for example, `// TODO-HIBERNATE-34`
- The description of the https://jira.mongodb.org/browse/HIBERNATE-34 ticket should mention that resolving all the `TODO-HIBERNATE-34` comments is in scope of the ticket.

I'll apply this approach to the `main` branch.

Reply:
Given Hibernate-34 has been merged, I made further code changes to parse the version text in `BuildConfig.VERSION` to populate the two driver version numbers, together with a unit testing case.

Not sure whether it is over-engineering, but it seems good move.